_translator-files/po/sv-SE/Installation-for-Windows.po

```diff
@@ -228,51 +234,55 @@ msgid ""
 "1. Headphones and speakers are often labelled as \"output\" and microphones as \"input\" or \"capture\".\n"
 "1. Stereo Mix/Stereo Input is usually not the input/output you're looking for. Therefore, switch it off if you see it.\n"
 msgstr ""
+"**Tips:**\n"
+"1. Det kan vara oklart vad den korrekta ljudenheten heter. Många interna ljudkort heter \"Realtek High Definition Audio\", \"Conexant\" eller liknande.\n"
+"1. Hörlurar och högtalare är ofta märkta som \"output\" och mikrofoner som \"input\" eller \"capture\".\n"
+"1. Stereo Mix/Stereo Input är vanligtvis inte den ingång/utgång du letar efter. Stäng därför av den om du ser den.\n"
 
 #. type: Title ###
 #: ../wiki/en/Installation-for-Windows.md:73
 #, markdown-text, no-wrap
 msgid "Troubleshooting"
-msgstr ""
+msgstr "Felsökning"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:76
 #, markdown-text
 msgid "If nothing works, first restart Jamulus and/or your PC to close background processes that may be accessing your sound card."
-msgstr ""
+msgstr "Om inget fungerar, starta först om Jamulus och/eller din dator för att stänga bakgrundsprocesser som kan ha åtkomst till ditt ljudkort."
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:78
 #, markdown-text
 msgid "Afterwards, *set up the inputs/outputs again*. Enabled and accessible input/outputs show as lit up power buttons and play buttons in the ASIO4ALL settings. If instead you see a red cross or yellow symbol, close other applications that may be accessing your sound card (e.g. web browser, Zoom, etc)."
-msgstr ""
+msgstr "Därefter, *ställ in ingångarna/utgångarna igen*. Aktiverade och tillgängliga ingångar/utgångar visas som upplysta strömknappar och uppspelningsknappar i ASIO4ALL-inställningarna. Om du istället ser ett rött kryss eller en gul symbol, stäng andra applikationer som kan ha åtkomst till ditt ljudkort (t.ex. webbläsare, Jitsi, Zoom, etc.)."
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:80
 #, markdown-text
 msgid "Have a look at [this video](https://youtu.be/_GzOsitVgLI) by [trombonepizza](https://github.com/trombonepizza) which gives more detailed setup information on ASIO4ALL."
-msgstr ""
+msgstr "Ta en titt på [denna video](https://youtu.be/_GzOsitVgLI) av [trombonepizza](https://github.com/trombonepizza) som ger mer detaljerad information om hur du ställer in ASIO4ALL."
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:82
 #, markdown-text
 msgid "Official and further information about how to configure ASIO4ALL is documented in the official [ASIO4ALL FAQs on the ASIO4ALL website](https://www.asio4all.org/index.php/help/faq/){: target=\"_blank\" rel=\"noopener noreferrer\"}."
```

Comment:
Would be worth updating the link https://www.asio4all.org/index.php/help/faq/ to https://asio4all.org/help/frequently-asked-questions/